docs/current_docs/cloud/572923-get-started.mdx

```diff
@@ -303,7 +303,7 @@ This section assumes that you have a Node.js development environment. It uses th
   ```javascript file=./snippets/get-started/appa/index.mjs
   ```
 
-  This Dagger pipeline uses the Dagger Node.js SDK to test, build and publish a containerized version of the application to a public registry.
+  This Dagger pipeline uses the Dagger Typescript SDK to test, build and publish a containerized version of the application to a public registry.
```

Comment:
@TomChv I saw that you changed the text here and in other places, but the snippets are still in Javascript. Is that correct? Shouldn't we also change each snippet to TypeScript as part of this diff?

Reply:
I could do that too that's write, I'm going to update that